CSharpSourceCode/Utilities/Extensions/AgentExtensions.cs

```diff
@@ -189,6 +193,12 @@ public static void ApplyDamage(this Agent agent, int damageAmount, Agent damager
                         blow.DamageType = DamageTypes.Invalid;
                         blow.VictimBodyPart = BoneBodyPartType.Chest;
                         blow.StrikeType = StrikeType.Invalid;
+
+                        if (damager == null)
+                        {
+                            TOWCommon.Say("damager not found");
+                        }
```

Comment:
null check is excess. TOWCommon.Say can be dropped down to place after line 216

Reply:
check I just deleted that line... was just for debug purpose